### PR TITLE
Global styles

### PR DESCRIPTION
**Pull request by Shazeen15**

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -22,12 +22,72 @@ Nav - header - #DAF2DC
...
+    display: block;
+    padding: 3%;
+    margin: 3%;
+  }
+
+  .card {
+    display: block;
+    padding: 3%;
+    box-shadow: 2px 2px 4px $light-blue;
+  }
```
Comment (minachuong) [inline app/assets/stylesheets/application.scss]: looks like .card is getting redefined

File: app/assets/stylesheets/yelpSearch.scss [hunk 2]
```diff
@@ -1,18 +1,5 @@
 @import "variables";
```
Comment (minachuong) [inline app/assets/stylesheets/yelpSearch.scss]: looks like variables import isn't being used in this file and can be removed

File: app/assets/stylesheets/yourTrips.scss [hunk 3]
```diff
@@ -1,43 +1 @@
 @import "variables";
```
Comment (minachuong) [inline app/assets/stylesheets/yourTrips.scss]: consider deleting the file if it doesn't have any component specific styles

File: app/javascript/components/pages/YourTrips.js [hunk 4]
```diff
@@ -11,28 +11,24 @@ export default class YourTrips extends Component {
         return trip.user_id === user.id;
       });
     return (
-      <div className="your-trips-container">
-        <h1 className="yt-header-title">Your Trips</h1>
-        <div className="yt-card">
+      <div className="main-card-container your-trips-container">
```
Comment (minachuong) [inline app/javascript/components/pages/YourTrips.js]:
```suggestion
      <div className="main-card-container">
```
 `your-trips-container` isn't being used

File: app/assets/stylesheets/tripshow.scss [hunk 5]
```diff
@@ -1,81 +1,85 @@
...
-     text-decoration: none;
-     color: $blue-purple;
+  .ts-btn {
+    box-shadow: 1px 1px !important;
+    background-color: $pink !important;
+    padding: 10px !important;
+    border-radius: 10px !important;
+    border: none !important;
+    margin: 3% !important;
+    color: $blue-purple !important;
```
Comment (minachuong) [inline app/assets/stylesheets/tripshow.scss:52]: `.ts-btn` looks like it has a lot of the same styles as `.delete-btn`, `.btd-btn` and `.edit-btn` with the exception of some colors. For duplicate CSS blocks consider using sass mixins which supports the use of parameters... same for `:hover` styling on the buttons

**Review by minachuong — APPROVED** [review]
no blockers but looks like there can be some refactoring to reduce duplicate code

File: app/assets/stylesheets/tripshow.scss (hunk 5 above)
Comment (Shazeen15) [reply app/assets/stylesheets/tripshow.scss:52]: going to come back to this after looking sass mixings

**Pull request merged**